### PR TITLE
[mig][14.0]product_supplierinfo_barcode

### PR DESCRIPTION
**Pull request by hkapatel-initos**

### CODE REVIEW

File: product_supplierinfo_barcode/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Barcode in supplier pricelist",
     "summary": "Add a barcode to supplier pricelist items",
-    "version": "12.0.1.1.0",
+    "version": "14.0.1.1.0",
```
Comment (rousseldenis) [inline product_supplierinfo_barcode/__manifest__.py]: Should be 14.0.1.0.0

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Review by fshah-initos — COMMENTED** [review]
LGTM

**Review by fshah-initos — APPROVED** [review]

**Comment by hkapatel-initos** [comment]
it would be nice if someone reviews here

**Comment by hkapatel-initos** [comment]
hello, @eLBati it would be nice if you review it here.

File: product_supplierinfo_barcode/__manifest__.py (hunk 1 above)
Comment (hkapatel-initos) [reply product_supplierinfo_barcode/__manifest__.py]: Change is done can you please approve this pull.

**Review by eLBati — APPROVED** [review]
LGTM.
@hkapatel-initos You already tested, right?

File: product_supplierinfo_barcode/__manifest__.py (hunk 1 above)
Comment (hkapatel-initos) [reply product_supplierinfo_barcode/__manifest__.py]: hello @rousseldenis it would be nice if you give approval here.

**Comment by hkapatel-initos** [comment]
hello @rousseldenis it would be nice if you give approval here.

**Comment by hkapatel-initos** [comment]
@eLBati Can we merge this?

**Comment by eLBati** [comment]
@rousseldenis could you update your review? Thanks

**Review by rousseldenis — APPROVED** [review]

**Comment by hkapatel-initos** [comment]
Can we merge this?

**Comment by hkapatel-initos** [comment]
Can we merge this?

**Pull request merged**